packages/manager/modules/support/src/support.controller.js

```diff
@@ -0,0 +1,42 @@
+import forEach from 'lodash/forEach';
+
+import { EVENTS } from './support.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $transitions,
+  ) {
+    this.$scope = $scope;
+    this.$transitions = $transitions;
+  }
+
+  $onInit() {
+    const transitionCriteria = {
+      from: 'support.**',
+    };
+
+    this.unsubscribers = [
+      this.$scope.$on(EVENTS.startLoading, () => this.startLoading()),
+      this.$scope.$on(EVENTS.stopLoading, () => this.stopLoading()),
+      this.$transitions.onStart(transitionCriteria, () => this.startLoading()),
+      this.$transitions.onSuccess(transitionCriteria, () => this.stopLoading()),
```

Comment:
Imho, this should be extracted from the `controller` as it is reliant on UI router

Reply:
Don't know how to do this outside the `controller`, I'd be glad to have your input on the matter 😃